src/screens/PostScreen.tsx

```diff
@@ -45,19 +76,102 @@ export default class PostScreen extends React.Component<any, PostState>{
       getCommetsFromPostId(this.state.postId).then((data) => {
         //@ts-ignore
         this.setState({comments : data});
+        //@ts-ignore
+        this.setState({post_hanesst_id: data[0].hanesst_id})
+        console.log(data[0].hanesst_id)
       }); 
+    
     }
+   
+  }
+
+
+
+  handleSubmit = async ( e: React.FormEvent<HTMLFormElement>):Promise<void> => {
+      e.preventDefault();
+
+      if(this.state.username|| this.state.pwdHash || this.state.post_text || this.state.postId || this.state.post_hanesst_id){
+         //@ts-ignore
+      postNewComment(this.state.username, this.state.pwdHash, this.state.post_text, this.state.post_hanesst_id, this.state.comment_hanesst_id ).then(res => {
+        console.log(res);
+        });
+        
+        alert("Your comment was successfully posted! Please refresh the page to see it!");
+      } else {
+        alert("Something went wrong! Did you fill out all the inputs?");
+      }
+     
   }
 
+  newComment(){
+    
+    return(
+      <div>
+      <h2>New Comment</h2>
+      <h4>Comment Text</h4>
+      <form  onSubmit={this.handleSubmit}>
+        <label htmlFor="test" >
+          <input  
+           type="text"
+           style={{width:"75%", height: "25px", lineHeight:"1.5em", marginBottom:"20px", display: "inline-block"}} 
+           value={this.state.post_text}
+           onChange={e => this.handleCommentInput(e)} />
+
+          <h4>Hanesst ID</h4>

```

Comment:
Shouldn't need hanesst

Reply:
Have made a temp. solution by setting the hanesst ID to 0, until we find a better solution, pref. being able to create post and comments with a helge_id/hanesst_id of "Null"